sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -747,9 +747,7 @@ object functions {
    */
   @scala.annotation.varargs
   def struct(cols: Column*): Column = {
```

Comment:
The documentation above needs to be updated and should specify what happens when the columns are unnamed.